style.css

```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
+    line-height: 70px;
+    letter-spacing: -1.87579px;
+    color: #19191B;
+}
+.first_foot_bnt{
+    color: white;
+    background-color:#5454D4 ;
+    padding: 18px 7px 16px 8px;
+    border: none;
+    border-radius: 10px 0px 0px 10px;
+    width: 160px;
+    font-size: 16px;
+    line-height: 21px;
+    text-align: center;
+    letter-spacing: -0.500211px;
+    font-family: 'DM Sans', sans-serif;
+}
+.second_foot_bnt{
+    font-size: 16px;
+    line-height: 21px;
+    text-align: center;
+    letter-spacing: -0.500211px;
+    color: #19191B;
+    font-family: 'DM Sans', sans-serif;
+    background-color: #F7F7FB; 
+    border: none;
+    border-radius: 0px 10px 10px 0;
+    width: 160px;
+    padding: 18px 7px 16px 8px;
+}
+.footer__card_container{
+    padding-top: 98px;
+    padding-left: 185px;
+}
+.footer__card{
+    display: flex;
+    justify-content: baseline;
+}
+.first__footer_card{
+    width: 330px;
+    height: 441px;
+    border: 1px solid black;
+    text-align: center;
+    background-color: #F8F8F8;
+    border-radius: 5px;
+    margin-right: 18px;
+}
+.name__footer_card{
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    letter-spacing: -0.562737px;
+    color: #696871;
+    padding-top: 14px;
+    padding-bottom: 7px;
+}
+.title_footer{
+    font-size: 60px;
+    line-height: 56px;
+    text-align: center;
+    letter-spacing: -1.03448px;
+    color: #1D293F;
+}  
+
+.about_footer{
+    font-size: 17px;
+    line-height: 50px;
+    text-align: center;
+    letter-spacing: -0.531474px;
+    color: #696871;
+    font-family: 'Mulish', sans-serif;
+    padding: 5px 106px 36px 106px;
+    width: 136px;
+}
+.first__footer_card.second{
+    background-color: #FF7143;
+    width: 376px;
+    height: 473px;
+    position: relative;
+    bottom: 20px;
+    
+}
+.name__footer_card2{
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    letter-spacing: -0.562737px;
+    color: rgba(255, 255, 255, 0.7);
+    padding-top: 30px;
+}
+.title_footer2{
+    font-size: 60px;
+    line-height: 56px;
+    text-align: center;
+    letter-spacing: -1.03448px;
+    color: #FFFFFF;
+    font-family: 'DM Sans', sans-serif;
+    padding: 8px 92px 1px 83px;
+}
+.about_footer2{
+    font-size: 17px;
+    line-height: 50px;
+    text-align: center;
+    letter-spacing: -0.531474px;
+    color: #FFFFFF;
+    padding: 0 126px 20px 119px;
+}
+.footer_month{
+    font-size: 28px;
+}
+.footer_card_btn1{
+    background-color:white ;
+    border: 1px solid #EAE9F2;
+    border-radius: 10px;
+    width: 285px;
+    height: 60px;
+    padding: 19px 42px;
+    color: #5454D4;
+    font-size: 17px;
+    line-height: 22px;
+    text-align: center;
+    letter-spacing: -0.531474px;
+    font-family: 'DM Sans', sans-serif;
+}
+.footer_card_btn2{
+    background: #9F3919;
+    border: none;
+    border-radius: 10px;
+    width: 305px;
+    height: 60px;
+    padding: 19px 34px 43px 37px;
+    color: white;
```

Comment:
you should write color in one style 
if write color as HEX, write it

Reply:
done